test/oats/docker-compose-gotestserver.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.9'
+
+services:
+  # Go based test server
+  testserver:
+    image: grcevski/tests:gotestserver
```

Comment:
Might be time to upload these to ghcr.io, I did this recently for one of our test images that needed to be updated. For example we can use a common path like this: ghcr.io/grafana/beyla/test/greeting-rust${TESTSERVER_IMAGE_SUFFIX}/0.0.2